kbfsedits/tlf_history.go

```diff
@@ -242,7 +242,17 @@ var filesToIgnore = map[string]bool{
 
 func ignoreFile(filename string) bool {
 	_, base := path.Split(filename)
-	return filesToIgnore[base] || strings.HasPrefix(base, "._")
+	if filesToIgnore[base] || strings.HasPrefix(base, "._") {
+		return true
+	}
+	// Treat the files to ignore as prefixes, since if they ever
+	// conflict they'll have the conflict suffix.
```

Comment:
Is there an easy way to check for conflict suffix to avoid false positives here?

Reply:
We could with regular expressions, but that seems too expensive.  I can't imagine a valid file with these prefixes.